.github/actions/setup-minikube/action.yaml

```diff
@@ -41,7 +41,8 @@ runs:
       - level: RequestResponse
       EOF
 
-      sudo minikube start --wait=all --extra-config='apiserver.event-ttl=24h' --extra-config='apiserver.audit-policy-file=/etc/ssl/certs/audit-policy.yaml' --extra-config='apiserver.audit-log-path=/var/log/kube-apiserver-audit.log'
+      # We are not using any addons, so we disable the installation altogether. If it was to be enabled, `storage-provisioner` and `default-storageclass` ought to be explicitly disabled.
```

Comment:
I added the other part of the comment, i.e. "If it was to be enabled, `storage-provisioner` and `default-storageclass` ought to be explicitly disabled." as a future reference. They are enabled by default, so if some wanted to enable addons, they would have to disable these two explicitly, otherwise it would break some stuff. I'll try to rewrite it in simpler words anyway though.